node/mock-run.ts

```diff
@@ -32,10 +35,43 @@ export class TaskMockRunner {
      *
      * @param answers   Answers to be returned when the task lib functions are called.
      */
-    public setAnswers(answers: ma.TaskLibAnswers) {
+    public setAnswers(answers: TaskLibAnswers) {
         this._answers = answers;
     }
 
+    checkModuleName(modName: any): boolean {
+        if (typeof modName !== 'string') {
+            return false;
+        }
+
+        if (modName.includes('.')) {
+            console.log(`WARNING: ${modName} is a local module. Cannot require it from task-lib. Please pass an already imported module.`);
+            return false;
+        }
+
+        return true;
+    }
+
+
+    checkIsMockable(newModule, methodName, oldModule) {
+        const method = newModule[methodName];
+
+        if (!newModule.hasOwnProperty(methodName) || typeof method === 'undefined') {
+            return false;
+        }
+
+        if (typeof method !== 'function') {
+            console.log(`WARNING: ${methodName} of ${newModule} is not a function. There is no option to replace getter/setter in this implementation. You can consider changing it.`);
```

Comment:
probably, [console.warn](https://nodejs.org/api/console.html#consolewarndata-args) (but nor sure if we want to redirect to stderr)